daprdocs/content/en/reference/components-reference/supported-state-stores/_index.md

```diff
@@ -26,38 +26,38 @@ The following stores are supported, at various levels, by the Dapr state managem
 
 ### Generic
 
-| Name                                                           | CRUD | Transactional | ETag | Actors | Status | Component version | Since |
-|----------------------------------------------------------------|------|---------------------|------|------|--------| -------|------|
-| [Aerospike]({{< ref setup-aerospike.md >}})                    | ✅   | ❌                  | ✅    | ❌ | Alpha  | v1 | 1.0 |
-| [Apache Cassandra]({{< ref setup-cassandra.md >}})             | ✅   | ❌                  | ❌    | ❌ | Alpha  | v1 | 1.0 |
-| [Cloudstate]({{< ref setup-cloudstate.md >}})                  | ✅   | ❌                  | ✅    | ❌ | Alpha  | v1 | 1.0 |
-| [Couchbase]({{< ref setup-couchbase.md >}})                    | ✅   | ❌                  | ✅    | ❌ | Alpha  | v1 | 1.0 |
-| [Hashicorp Consul]({{< ref setup-consul.md >}})                | ✅   | ❌                  | ❌    | ❌ | Alpha  | v1 | 1.0 |
-| [Hazelcast]({{< ref setup-hazelcast.md >}})                    | ✅   | ❌                  | ❌    | ❌ | Alpha  | v1 | 1.0 |
-| [Memcached]({{< ref setup-memcached.md >}})                    | ✅   | ❌                  | ❌    | ❌ | Alpha  | v1 | 1.0 |
-| [MongoDB]({{< ref setup-mongodb.md >}})                        | ✅   | ✅                  | ✅    | ✅ | GA  | v1 | 1.0 |
-| [MySQL]({{< ref setup-mysql.md >}})                            | ✅   | ✅                  | ✅    | ✅ | Alpha  | v1 | 1.0 |
-| [PostgreSQL]({{< ref setup-postgresql.md >}})                  | ✅   | ✅                  | ✅    | ✅ | Alpha  | v1 | 1.0 |
-| [Redis]({{< ref setup-redis.md >}})                            | ✅   | ✅                  | ✅    | ✅ | GA  | v1 | 1.0 |
-| [RethinkDB]({{< ref setup-rethinkdb.md >}})                                                      | ✅   | ✅                  | ✅    | ✅ | Alpha  | v1 | 1.0 |
-| [Zookeeper]({{< ref setup-zookeeper.md >}})                    | ✅   | ❌                  | ✅    | ❌ | Alpha  | v1 | 1.0 |
+| Name                                                           | CRUD | Transactional | ETag | TTL | Actors | Status | Component version | Since |
```

Comment:
While we're doing this PR, could you also make the headers hyperlinks to the sections about the feature? For example TTL can link to the docs on TTL and Actors could link to the Actors docs

Reply:
Done now (I think)